features/frameworks/greentea-client/source/greentea_test_env.cpp

```diff
@@ -562,7 +557,21 @@ enum Token {
  *
  */
 extern "C" int greentea_getc() {
-    return greentea_serial->getc();
+    uint8_t c;
+    read(STDERR_FILENO, &c, 1);
```

Comment:
We should use STDOUT_FILENO instead of STDERR_FILENO.

Reply:
Quite right - that `STDERR` is just because it was copied from `mbed_error`.

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/037924aeeb7c93705576fa38b71b19fa5db44bcd..d7f555c28159bf53cdbfb925383b5dc5ae6a47b3).